integrations/tensorflow/test/.gitignore

```diff
@@ -0,0 +1,2 @@
+.lit_test_times.txt
```

Comment:
Can we make the test execution directory the temp directory so that lit doesn't write random files here? I quite dislike that it does this

```python
config.test_exec_root = os.path.join(tempfile.gettempdir(), "lit")
```

Reply:
Done.